test/db/cmd/types

```diff
@@ -1752,7 +1791,6 @@ EXPECT=<<EOF
 var char ** var_14e0h @ rbp-0x14e0
 var int var_14d4h @ rbp-0x14d4
 var int * wstatus @ rbp-0x14cc
-var int * options @ rbp-0x14c8
```

Comment:
why this change? is this a regression?

Reply:
It's 64bit binary, pointer length detected as 8 bytes, thus this variable is overlapped with the previous `wstatus`.